doc/reference/devices_nic.md

```diff
@@ -442,6 +442,7 @@ Key                     | Type    | Default           | Description
 `limits.egress`         | string  | -                 | I/O limit in bit/s for outgoing traffic (various suffixes supported, see {ref}`instances-limit-units`)
 `limits.ingress`        | string  | -                 | I/O limit in bit/s for incoming traffic (various suffixes supported, see {ref}`instances-limit-units`)
 `limits.max`            | string  | -                 | I/O limit in bit/s for both incoming and outgoing traffic (same as setting both `limits.ingress` and `limits.egress`)
+`limits.priority`       | integer | -                 | `skb->priority` (32-bit unsigned integer) for outgoing traffic. This value is used by the kernel queueing discipline (qdisc) to prioritize network packets. Effect of this value depending on a particular qdisc implementation (`SKBPRIO`, `QFQ`, etc). You need to consult with the kernel qdisc documentation before setting anything here.
```

Comment:
```suggestion
`limits.priority`       | integer | -                 | The `skb->priority` value (32-bit unsigned integer) for outgoing traffic, to be used by the kernel queuing discipline (qdisc) to prioritize network packets (The effect of this value depends on the particular qdisc implementation, for example, `SKBPRIO` or `QFQ`. Consult the kernel qdisc documentation before setting this value.)
```
Trying to fit this into the existing format - this will work much better when we move this to the code and have more space for explanations. :)

Reply:
Thanks a lot! Fixed!